### ISSUE TITLE
Need a way to avoid reorganizing structs that are marshaled

### ISSUE THREAD

**Issue by glen-nicol** [codecadwallader/codemaid#530]
### Environment
- Visual Studio version: 2013 professional
- CodeMaid version: 10.4
- Code language: C#

### Description
I am hesitant to use the global reorganization feature without some way to ensure that my structs are not touched because their layout matches native structs. Could the structlayout attribute be used to detect this and disable reorganization? Or could the Types setting allow a type to be removed from the order so it is not reorganized ever?

**Comment by codecadwallader** [comment]
Yes, we already do look for the `[StructLayout]` attribute and ignore it for reorganization purposes.

https://github.com/codecadwallader/codemaid/blob/master/CodeMaid/Logic/Reorganizing/CodeReorganizationManager.cs#L221

Please give it a shot at small scale and confirm it works as expected for you!

**Comment by glen-nicol** [comment]
Awesome! It does work as you say. I didn't see that documented anywhere though. Might be worthwhile to mention that in the UI.

**Comment by codecadwallader** [comment]
Thanks for the suggestion.  Documentation isn't something we've done great about, but perhaps it would fit under here: http://www.codemaid.net/documentation/#reorganizing
